telegram/_chatfullinfo.py

```diff
@@ -44,6 +44,9 @@ class ChatFullInfo(_ChatBase):
 
     .. versionadded:: 21.2
 
+    .. versionchanged:: NEXT.VERSION
+        You can now also use the shortcut methods of :class:`telegram.Chat` on this object.
+
```

Comment:
these are already available right now b/c CFI is a subclass of Chat … At least the wording is a bit confusing to me. Maybe

```suggestion
    .. versionchanged:: NEXT.VERSION
        Explicit support for all shortcut methods known from :class:`telegram.Chat` on this object. Previously those were only available bcause this class inherited from :class:`telegram.Chat`.
```
?